stable/elastic-stack/README.md

```diff
@@ -20,7 +20,7 @@ This chart will do the following:
 To install the chart with the release name `my-release`:
 
 ```bash
-$ helm install --name my-release stable/elastic-stack
+$ RELEASE_NAME=my-release helm install --set kibana.env.ELASTICSEARCH_URL=http://${RELEASE_NAME}-elasticsearch-client:9200 --name ${RELEASE_NAME} stable/elastic-stack
```

Comment:
Can't we find a way to integrate this in the YAML by using default values instead of increasing the complexity of the install command?

Reply:
Ideally yes, but as spoken in issue #11037 I have no idea how to do it. Any suggestions?